port/port_posix.h

```diff
@@ -11,9 +11,11 @@
 
 #pragma once
 
+#include <cassert>
```

Comment:
Yeah, there are a few files affected. Ok, I will try to include headers in those files, instead of putting it here. I was unsure which way is better, so put a simple hack here.